web/src/routes/search/containers/__tests__/SearchPage.spec.js

```diff
@@ -157,4 +157,30 @@ describe('SearchPage', () => {
       location
     })
   })
+
+  describe('Tests for url query', () => {
+    it('should set state from url', () => {
+      const location = createLocation({
+        type: SEARCH_ROUTE, payload: { city, language }, query: { query: 'SearchForThis' }
+      })
+      const searchPage = shallow(
```

Comment:
We want to convert our tests from using enzyme to those using the react testing library: https://testing-library.com/docs/react-testing-library/intro/. This is fine for now, in the future best use this library though.